libs/cosmos-sdk/x/params/types/errors.go

```diff
@@ -12,4 +12,5 @@ var (
 	ErrEmptySubspace    = sdkerrors.Register(ModuleName, 4, "parameter subspace is empty")
 	ErrEmptyKey         = sdkerrors.Register(ModuleName, 5, "parameter key is empty")
 	ErrEmptyValue       = sdkerrors.Register(ModuleName, 6, "parameter value is empty")
+	ErrUpgradeNotReady  = sdkerrors.Register(ModuleName, 7, "no upgrade callback is registed")
```

Comment:
Does ErrUpgradeNotReady has not been used?

Reply:
No, it's useless now, I'll delete it.